site/content/docs/5.2/layout/grid.md

```diff
@@ -97,12 +97,12 @@ As noted above, each of these breakpoints have their own container, unique class
     <tbody>
       <tr>
         <th class="text-nowrap" scope="row">Container <code class="fw-normal">max-width</code></th>
-        <td>320px</td>
-        <td>480px</td>
-        <td>768px</td>
-        <td>1024px</td>
-        <td>1280px</td>
-        <td>1440px</td>
+        <td>302px</td>
```

Comment:
Are you sure those are the right values? It is mentioned that they must be the `max-width`.

Reply:
You're right, it should be the same as mentioned in containers ! Changing them.